### PR TITLE
correct description of secret usage

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@csrwng @nhr

**Comment by nhr** [comment]
Looks fine. @csrwng ?

File: using_openshift/builds.adoc [hunk 1]
```diff
@@ -51,7 +51,7 @@ Webhook triggers allow you to trigger a new build by sending a request to the Op
 * Generic Webhooks
 
 ==== Github webhooks
-Github webhooks can handle the call made by Github when a repository is updated.  When defining the trigger you can specify a `secret` which you will supply to Github when configuring the webhook.  This will ensure that only you and your repository can trigger the build.  The trigger definition json within the BuildConfig looks like this:
+Github webhooks can handle the call made by Github when a repository is updated.  When defining the trigger you can specify a `secret` which you will be a part of the URL you supply to Github when configuring the webhook.  This will ensure that only you and your repository can trigger the build.  The trigger definition json within the BuildConfig looks like this:
```
Comment (csrwng) [inline using_openshift/builds.adoc]: > which you will be a part of the URL you ...

which will be part of the URL you ...

**Comment by bparees** [comment]
sigh. fixed, thanks @csrwng

**Pull request merged**